src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -270,6 +281,18 @@ private void styleFetchedListView(CheckListView<CitationRelationItem> listView)
         listView.setSelectionModel(new NoSelectionModel<>());
     }
 
+    private void openDuplicateEntriesWindow(BibEntry entry) {
+        Dialog<ButtonType> dialog = new Dialog<>();
+        dialog.setTitle(Localization.lang("Possible duplicate entries"));
+
+        DialogPane dialogPane = new DialogPane();
+        dialogPane.setContent(new Label("Here you can show the duplicate entries and their comparison details."));
```

Comment:
Still no localization - see https://devdocs.jabref.org/code-howtos/localization.html for details.